### DISCUSSION TITLE
Ignore LiquidHTMLParsingError

### DISCUSSION THREAD

**Discussion by modulareverything** [Shopify/prettier-plugin-liquid#183] [Q&A] [answered]
Is there any way I can ignore this rule? We have a block that, depending on the number of blocks within it, will conditionally render either a swiper or just a regular layout. The problem is it means closing an if statement before we close the ending </div> as the code will look something like this:

```liquid
{%- if section.blocks.size > 1 -%}
      <div class="swiper">
{%- else -%}
      <div class="not-a-swiper">
{%- endif -%}
            The rest of the content
</div>
```

So it's weird-looking code but it works. Unfortunately though this plugin throws an error:

> `Attempting to close LiquidTag 'if' before HtmlElement 'div' was closedprettier-plugin-liquid(LiquidHTMLParsingError)`

and so the formatting doesn't work for the rest of the code. I tried using this:

`{% comment %}theme-check-disable LiquidHTMLParsingError{% endcomment %}`

but it doesn't actually seem to work in this case (a bug? maybe I wrote it wrong?)

**Comment by charlespwd** [comment, marked as answer]
:wave: You can check this issue for more context:
https://github.com/Shopify/prettier-plugin-liquid/issues/90

It's still on our radar, but got to juggle priorities. A solution for this is coming...
